### DISCUSSION TITLE
Remove `namespace` from k8s event exporter kustomization

### DISCUSSION THREAD

**Discussion by dmsolr** [apache/skywalking#7705] [Ideas]
Now, k8s-event-exporter's kustomization managed `namespace` resource. using `kustomize build ./release | kubectl delete -f -` to deployment will delete the namespace within all resources If users deployed it in their namespace.

That sounds not good. So I propose that we don't manage the namespace. If the namespace does not exist, give a tip for the user on how to create a namespace.

**Comment by kezhenxu94** [comment]
Good idea, are you going to open a pull request?
